plugins/out_s3/s3.c

```diff
@@ -1447,6 +1529,18 @@ static struct flb_config_map config_map[] = {
      0, FLB_FALSE, 0,
     "Predefined Canned ACL policy for S3 objects."
     },
+    {
+     FLB_CONFIG_MAP_STR, "compression", NULL,
+     0, FLB_FALSE, 0,
+    "Compression type for S3 objects. 'gzip' is currently the only supported value. "
+    "This will be set as the Content-Encoding HTTP header."
```

Comment:
Just to be very clear, instead of saying "This will be set", say, "The Content-Encoding HTTP Header will be set to 'gzip'"